bindings/ruby/bin/rock-gazebo-viz

```diff
@@ -54,7 +59,19 @@ sdf.each_model(recursive: true) do |model|
     model_viz.frame = model.name
 
     task_name = "gazebo:#{model.full_name.gsub('::', ':')}"
-    task_proxy = Orocos::Async.proxy task_name
```

Comment:
This change is a bad idea. The use of Async.proxy makes sure that we can start the script even if gazebo is not running. What you do here breaks this property, which is critical for every vsualization script. Moreover, if you tear down gazebo and start it again, it won't work as expected.

This is a visualization script, not a runtime script. Do NOT change the task's configuration in any way here. If you need to, it has to be done in your system's runscripts (e.g. syskit) or in another startup script.